doc/user_guide/eds.rst

```diff
@@ -525,31 +538,53 @@ Quantification can be applied from the intensities (background subtracted) with
 
 .. code-block:: python
 
     >>> # With s, intensities and kfactors from before
-    >>> s.quantification(intensities, kfactors, plot_result=True,
-    >>>                  composition_units='atomic')
-    Fe (Fe_Ka): Composition = 15.41 atomic percent
-    Pt (Pt_La): Composition = 84.59 atomic percent
+    >>> s.quantification(intensities, method='CL',kfactors,
+    >>>                  composition_units='weight')
+    Fe (Fe_Ka): Composition = 4.96 weight percent
+    Pt (Pt_La): Composition = 95.04 weight percent
 
-either with :py:func:`~.misc.material.weight_to_atomic`. The reverse method is :py:func:`~.misc.material.atomic_to_weight`.
+or with :py:func:`~.misc.material.atomic_to_weight`. The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
 
 .. code-block:: python
 
-    >>> # With weight_percent from before
-    >>> atomic_percent = hs.material.weight_to_atomic(weight_percent)
+    >>> # With atomic_percent from before
+    >>> weight_percent = hs.material.atomic_to_weight(atomic_percent)
+
+The zeta-factor method needs both the beam_current (in nA) and the acquisition or dwell time (referred to as real_time in seconds) in order to provide an accurate quantification. Both of the these parameters can be assigned to the metadata of the spectrum using:
+
+..code-block:: python
+
+    >>> s.set_microscope_parameters(beam_current = 0.5)
+    >>> s.set_microscope_parameters(real_time = 1.5)
+
+If these are not set the code will produce an error stating which parameter has been forgotten.
+The zeta-factor method will produce two sets of results. Index [0] is the composition maps for each element in atomic percent and index [1] is the mass-thickness map.
+
+The cross section method needs the beam_current, dwell time (real_time) and probe area in order to provide accurate quantification. The beam_current and real_time can be set using :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.set_microscope_parameters` as shown above. The beam_area (in nm^2) can be defined in two different ways. Either, the probe diameter is narrower than the pixel width, which case the probe is being under-sampled and therefore an estimation of the probe area needs to be used. This can be added to the metadata by:
+
+..code-block: python
+
+    >>> s.set_microscope_parameters(beam_area = 0.00125)
+
+ Alternatively, if sub-pixel scanning is used (or the spectrum map was recorded at a high spatial sampling and then the data binned into much larger pixels before quantification) the illumination area then becomes the pixel area of the spectrum image. This is a much more accurate approach for quantitative EDX and should be used where possible.  The pixel width could either be added to the metadata by putting the pixel area in as the beam_area (above) or by calibrating the spectrum image (see `Setting axis properties <http://hyperspy.org/hyperspy-doc/dev/user_guide/getting_started.html#setting-axis-properties>`_ )
```

Comment:
Internal links should use labels instead of the full address. See http://www.sphinx-doc.org/en/stable/markup/inline.html#ref-role